_includes/footer_legal.html

```diff
@@ -1,4 +1,4 @@
-&copy; {{ 'now' | date: "%Y" }} <a href="https://aws.amazon.com/"> Amazon Web Services</a> and individual contributors. OpenSearch is a <a href="/trademark-usage.html">registered trademark</a> of Amazon Web Services.</a> <br /><br />
+&copy; OpenSearch contributors, {{ 'now' | date: "%Y" }}. OpenSearch is a <a href="/trademark-usage.html">registered trademark</a> of Amazon Web Services.</a> <br /><br />
```

Comment:
Did we capitalize Contributors elsewhere?

Reply:
It was in quotes from legal, so I took it as-is.